modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/service/impl/CPDefinitionLinkLocalServiceImpl.java

```diff
@@ -75,6 +95,29 @@ public CPDefinitionLink addCPDefinitionLinkByCProductId(
 
 		User user = _userLocalService.getUser(serviceContext.getUserId());
 
+		Date expirationDate = null;
+		Date now = new Date();
```

Comment:
I've just seen this https://github.com/brianchandotcom/liferay-portal/pull/134941#issuecomment-1553691954 this should be calld date